Dockerfile

```
@@ -28,6 +28,7 @@ COPY gunicorn.conf.py gunicorn.conf.py
 COPY host_reaper.py host_reaper.py
 COPY host_synchronizer.py host_synchronizer.py
 COPY inv_mq_service.py inv_mq_service.py
+COPY inv_publish_hosts.py inv_publish_hosts
```

```suggestion
COPY inv_publish_hosts.py inv_publish_hosts.py
```
Minor typo here. You can see the job is still currently failing in the tests.